bricks/containers/src/grid-layout/index.tsx

```
@@ -0,0 +1,282 @@
+    }
+  }
+
+  handleShowGridBorder(): void {
+    if (this.showGridBorder) {
+      this.style.setProperty(
+        "--grid-border-color",
+        this.gridBorderColor ?? "#454547"
+      );
+    } else {
+      if (this.style.getPropertyValue("--grid-border-color")) {
+        this.style.removeProperty("--grid-border-color");
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)